backend/src/data/repositories/user/user.repository.ts

```diff
@@ -12,11 +12,11 @@ class User {
     this.#UserModel = UserModel;
   }
 
-  getAll(): Promise<TUser[]> {
+  public getAll(): Promise<TUser[]> {
     return this.#UserModel.query();
   }
 
-  create(payload: UserCreatePayload): Promise<TUser> {
+  public create(payload: UserCreatePayload): Promise<TUser> {
```

Comment:
I would suggest not specifying a public access modifier. Since the absence of an access modifier in the typescript means that access to the class methods is public.